pkg/apis/eksctl.io/v1alpha5/iam.go

```diff
@@ -25,6 +25,8 @@ type ClusterIAM struct {
 	// +optional
 	WithOIDC *bool `json:"withOIDC,omitempty"`
 	// +optional
+	OIDCClientIDList []string `json:"oidcClientIDList,omitempty"`
```

Comment:
To follow the same convention of the other fields I would use `OIDCClientIDS` 
(Remember to update https://github.com/weaveworks/eksctl/pull/1788/files#diff-4572c83eda74aa47ee807019c6900162R45)

Reply:
I will use `OIDCClientIDs`